### PR TITLE
Updates submodules for ion-functions

### PR DESCRIPTION
**Pull request by lukecampbell**
Previous HEAD position was 8a8b432... Merge pull request #68 from
lukecampbell/loclrng-r2
HEAD is now at 8aae5b4... Merge pull request #69 from
lukecampbell/fillvals-r2

### CODE REVIEW

**Comment by lukecampbell** [comment]
@jamie-cyber1 Ready to merge at your leisure.

File: extern/marine-integrations [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 99a14c98ccb82ac41b879ad1ad7d37d2f7354e24
```
Comment (jamie-cyber1) [inline extern/marine-integrations]: Why mi pointer here?

Comment (lukecampbell) [inline extern/marine-integrations]: That's an EXCELLENT question.

**Comment by lukecampbell** [comment]
So, when I updated the submodule I did a `git add extern/` before committing and apparently we don't have a marine-integrations submodule anymore but I still have a folder (with a git reference) so it got roped in with the commit, I just force-pushed a proper commit and it is ready to be reviewed.

**Comment by jamie-cyber1** [comment]
Looks good.

**Pull request merged**